.github/workflows/ci.yml

```diff
@@ -19,20 +16,13 @@ jobs:
     - name: Setup Java JDK
       uses: actions/setup-java@v1.4.3
       with:
-        java-version: 11
-
-    - name: Build with Maven on master branch
-      if: github.ref == 'refs/heads/master'
-      run: mvn clean deploy --batch-mode
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        java-version: 8
 
-    - name: Build with Maven on non-master branch
-      if: github.ref != 'refs/heads/master'
+    - name: Build with Maven
       run: mvn clean verify --batch-mode
 
     - name: Publish Unit Test Results
       uses: EnricoMi/publish-unit-test-result-action@v1
       if: always()
       with:
-        files: "*/target/surefire-reports/*.xml"
\ No newline at end of file
+        files: "target/surefire-reports/*.xml"
```

Comment:
was the beginning asteric necessary to match reports in sub-modules?

Reply:
you are right. thanks. eagle eye.